tangram_bundler/__init__.py

```diff
@@ -25,25 +44,17 @@ def addDependencies(file_list, filename):
 
     # Search for textures urls
     if 'textures' in yaml_file:
-        for font in yaml_file['textures']:
```

Comment:
we still need the fonts thing until it's made optional, no?

Reply:
Oh, this old code is all matters of awkward.

Reply:
Ohh I just renamed `font` to `texture`